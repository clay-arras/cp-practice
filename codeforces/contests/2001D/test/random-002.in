1
14
19 10 7 19 15 5 14 1 15 17 9 15 9 25
