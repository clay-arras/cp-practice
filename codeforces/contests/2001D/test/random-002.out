10
19 10 7 5 14 1 17 9 15 25
